control-plane/csi-driver/Cargo.toml

```diff
@@ -35,6 +35,7 @@ rpc = { path = "../../rpc" }
 tokio = { version = "1.20.1", features = ["full"] }
 tokio-stream = { version = "0.1.7", features = ["net"] }
 clap = "2.33.3"
+nix = "0.24.2"
```

Comment:
Please either change this to "0.22.3" or change nvmeadm to use "0.24.2" to avoid referencing 2 different crate versions.